### PR TITLE
set right config values

### PR DESCRIPTION
**Pull request by mdemri**

### CODE REVIEW

File: worker/config/p2p/testnet_config.json [hunk 1]
```diff
@@ -1,28 +1,29 @@
...
+    "CONTRACTS_FOLDER": "/root/p2p/contracts/",
+    "KEY_MANAGEMENT_ADDRESS": "http://km-testnet.enigma.co:3040",
+    "KEY_MANAGEMENT_DISCOVERY": "http://km-service:8081",
+    "FAUCET_URL": "",
+    "MINIMUM_ETHER_BALANCE": "0.1",
+    "MINIMUM_ENG_BALANCE": "100000000000",
+    "DEPOSIT_AMOUNT": 100000000000,
+    "ETH_KEY_PATH": "/root/.enigma/",
+    "STAKE_KEY_PATH": "/root/.enigma/staking/",
+    "BOOTSTRAP_ID": "B1",
```
Comment (Cashmaney) [inline worker/config/p2p/testnet_config.json:16]: Don't need any BOOTSTRAP_ for testnet

File: worker/config/p2p/testnet_config.json [hunk 2]
```diff
@@ -1,28 +1,29 @@
...
+{
+    "CONTRACT_DISCOVERY_ADDRESS": "http://contract:8081",
+    "ETH_NODE_ADDRESS": "http://parity.enigma.co:8545",
+    "ENIGMA_CONTRACT_FILE_NAME": "Enigma.json",
+    "CORE_ADDRESS": "localhost",
+    "CORE_PORT": "5552",
+    "CONTRACTS_FOLDER": "/root/p2p/contracts/",
+    "KEY_MANAGEMENT_ADDRESS": "http://km-testnet.enigma.co:3040",
+    "KEY_MANAGEMENT_DISCOVERY": "http://km-service:8081",
+    "FAUCET_URL": "",
```
Comment (Cashmaney) [inline worker/config/p2p/testnet_config.json:10]: Don't need this

File: worker/config/p2p/testnet_config.json [hunk 3]
```diff
@@ -1,28 +1,29 @@
...
-}
+{
+    "CONTRACT_DISCOVERY_ADDRESS": "http://contract:8081",
+    "ETH_NODE_ADDRESS": "http://parity.enigma.co:8545",
+    "ENIGMA_CONTRACT_FILE_NAME": "Enigma.json",
+    "CORE_ADDRESS": "localhost",
+    "CORE_PORT": "5552",
+    "CONTRACTS_FOLDER": "/root/p2p/contracts/",
+    "KEY_MANAGEMENT_ADDRESS": "http://km-testnet.enigma.co:3040",
+    "KEY_MANAGEMENT_DISCOVERY": "http://km-service:8081",
```
Comment (Cashmaney) [inline worker/config/p2p/testnet_config.json:9]: Don't need this

File: worker/config/p2p/testnet_config.json [hunk 4]
```diff
@@ -1,28 +1,29 @@
...
+    "MINIMUM_ETHER_BALANCE": "0.1",
+    "MINIMUM_ENG_BALANCE": "100000000000",
+    "DEPOSIT_AMOUNT": 100000000000,
+    "ETH_KEY_PATH": "/root/.enigma/",
+    "STAKE_KEY_PATH": "/root/.enigma/staking/",
+    "BOOTSTRAP_ID": "B1",
+    "BOOTSTRAP_PORT": "10300",
+    "BOOTSTRAP_PATH": "/root/p2p/scripts/bootstrap/",
+    "LIBP2P_KEY_PATH": "/root/.enigma/libp2p/",
+    "BALANCE_WAIT_TIME": 1800,
```
Comment (Cashmaney) [inline worker/config/p2p/testnet_config.json:20]: Don't need this

File: worker/config/p2p/testnet_config.json [hunk 5]
```diff
@@ -1,28 +1,29 @@
...
+    "MIN_CONFIRMATIONS": 3,
+    "KEY_MANAGEMENT_TIMEOUT": 120,
```
Comment (Cashmaney) [inline worker/config/p2p/testnet_config.json:22]: Don't need this

File: worker/config/p2p/testnet_config.json [hunk 6]
```diff
@@ -1,28 +1,29 @@
...
+    "CONTRACT_TIMEOUT": 3600,
```
Comment (Cashmaney) [inline worker/config/p2p/testnet_config.json:23]: Don't need this

File: salad/operator/config/testnet_config.json [hunk 7]
```diff
@@ -1,11 +1,13 @@
...
+    "ETH_NODE_ADDRESS": "parity.enigma.co",
+    "ETH_NODE_PORT": "8545",
+    "ENG_NODE_ADDRESS": "worker",
+    "ENG_NODE_PORT": "3346",
+    "MONGO_URL": "mongodb://mongo:27017",
+    "DB_NAME": "salad",
+    "SECRET_CONTRACT_BUILD_FOLDER": "..",
+    "SALAD_SMART_CONTRACT_ADDRESS": "0x3ad332C3098C04bc60C23c7C469bBF19590C7848",
+    "SALAD_SECRET_CONTRACT_ADDRESS": "0x6bb065c82b68327ed064c4974211ede0f568517db84dc95aa77a274a9c2cfb29",
+    "KEY_MANAGEMENT_DISCOVERY": "THIS IS REDUNDANT FOR SALAD OPERATOR, REMOVE IN THE FUTURE"
```
Comment (Cashmaney) [inline salad/operator/config/testnet_config.json:12]: Don't need this anymore

File: salad/operator/config/k8s_config.json [hunk 8]
```diff
@@ -1,11 +1,11 @@
...
+ "ETH_NODE_ADDRESS": "contract-service",
  "ETH_NODE_PORT": "9545",
- "ENG_NODE_ADDRESS": "worker",
+ "ENG_NODE_ADDRESS": "bootstrap-1-service",
  "ENG_NODE_PORT": "3346",
  "MONGO_URL": "mongodb://mongo:27017",
  "DB_NAME": "salad",
  "SECRET_CONTRACT_BUILD_FOLDER": "..",
- "KEY_MANAGEMENT_DISCOVERY": "THIS IS REDUNDANT FOR SALAD OPERATOR, REMOVE IN THE FUTURE"
+ "KEY_MANAGEMENT_DISCOVERY": "unused"
```
Comment (Cashmaney) [inline salad/operator/config/k8s_config.json:10]: Don't need this anymore

**Review by Cashmaney — CHANGES_REQUESTED** [review]

File: salad/operator/config/testnet_config.json (hunk 7 above)
Comment (mdemri) [reply salad/operator/config/testnet_config.json:12]: what exactly is not needed? only KEY_MANAGEMENT_DISCOVERY?

File: worker/config/p2p/testnet_config.json (hunk 6 above)
Comment (mdemri) [reply worker/config/p2p/testnet_config.json:23]: what exactly is not needed?

File: worker/config/p2p/testnet_config.json (hunk 5 above)
Comment (mdemri) [reply worker/config/p2p/testnet_config.json:22]: what exactly is not needed?

File: worker/config/p2p/testnet_config.json (hunk 4 above)
Comment (mdemri) [reply worker/config/p2p/testnet_config.json:20]: what exactly is not needed?

File: worker/config/p2p/testnet_config.json (hunk 3 above)
Comment (mdemri) [reply worker/config/p2p/testnet_config.json:9]: what exactly is not needed?

File: worker/config/p2p/testnet_config.json (hunk 2 above)
Comment (mdemri) [reply worker/config/p2p/testnet_config.json:10]: what exactly is not needed?

File: salad/operator/config/k8s_config.json (hunk 8 above)
Comment (mdemri) [reply salad/operator/config/k8s_config.json:10]: KEY_MANAGEMENT_DISCOVERY?

File: worker/config/p2p/testnet_config.json (hunk 1 above)
Comment (mdemri) [reply worker/config/p2p/testnet_config.json:16]: only BOOTSTRAP_ID?

Comment (Cashmaney) [reply worker/config/p2p/testnet_config.json:16]: Anything necessary for the bootstraps (only BOOTSTRAP_ID in the case of testnet) should be set from the kubernetes deployment

File: worker/config/p2p/testnet_config.json (hunk 2 above)
Comment (Cashmaney) [reply worker/config/p2p/testnet_config.json:10]: All the comments refer to just the line above. In this case FAUCET_URL

**Pull request merged**